src/data/faq_de.json

```diff
@@ -3189,10 +3189,15 @@
                 }
             ],
             "P": [
+                {
+                    "term": "PCR-Schnelltest",
+                    "anchor": "rapid_pcr_test",
+                    "description": "PCR-Schnelltests, die oft auch als \"PoC-NAT-Tests\" bezeichnet werden, basieren auf der selben Technik wie reguläre <a href='#glossary_pcr_test'>PCR Tests</a>, lassen sich allerdings schnell und vor Ort auswerten und sind demnach besonders gut in Situationen geeignet, in denen ein relativ sicheres Testergebnis innerhalb kurzer Zeit benötigt wird. Im Gegensatz zu normalen PCR Tests sind sie in der Anwendung nicht auf ein medizinisches Labor beschränkt. Zum momentanen Zeitpunkt ist es nicht möglich ein PCR-Schnelltest Ergebnis in der Corona-Warn-App zu erhalten, die technischen Vorkehrungen wurden allerdings bereits in Version 2.19 implementiert."

```

Comment:
Please use "derselben" as a single instead of "der selben"

Reply:
Thanks for pointing that out. It has been changed in https://github.com/corona-warn-app/cwa-website/pull/2863/commits/8bfbbbe1adecd24b3108f26cffde3cea25d6459f